examples/namespaced/kustomization.yaml

```diff
@@ -0,0 +1,61 @@
+# This kustomize file can be used as an example for deploying Contour without ClusterRole and
```

Comment:
Noting that this does not work with Gateway API, since that requires watching Namespaces and GatewayClasses which are cluster-scoped. I think we can address this (if needed) along with the follow-up for the Gateway provisioner noted separately.

Reply:
Indeed, this kustomization file is now focusing on the "standard" Contour configuration (render/contour.yaml), which does not include Gateway API configuration. We can create a new kustomization file for namespaced Gateway-enabled Contour separately.